lua/graph.lua

```diff
@@ -1460,17 +1460,18 @@ function filter_tags_generic(kv)
   kv["alt_name"] = kv["alt_name"]
   kv["official_name"] = kv["official_name"]
 
-  if kv["maxspeed"] == "none" then
-    --- special case unlimited speed limit (german autobahn)
-    kv["max_speed"] = "unlimited"
-  else
-    kv["max_speed"] = normalize_speed(kv["maxspeed"])
+  -- max speeds are allowed to be unlimited
+  local speeds_tags = { max_speed="maxspeed", forward_speed="maxspeed:forward", backward_speed="maxspeed:backward" }
```

Comment:
I think it'd be more consistent to also allow the directional variants to be unlimited.. Shouldn't really happen, but I've seen it in at least synthetic OSM data